spec.html

```diff
@@ -8815,6 +8852,41 @@ <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _protot
       </emu-alg>
       <p>Each built-in function defined in this specification is created by calling the CreateBuiltinFunction abstract operation.</p>
     </emu-clause>
+
+    <emu-clause id="sec-createbuiltingeneratorinstance" aoid="CreateBuiltinGeneratorInstance">
+      <h1>CreateBuiltinGeneratorInstance ( _closure_ [ , _generatorBrand_ [ , _generatorPrototype_ [ , _generatorSlots_ ] ] ] )</h1>
```

Comment:
In the range proposal, I need to add extra slots on the iterator object because there is some helper methods on the %RangeIteratorPrototype% that need to access these internal variables but I can't access them out of the AbstractClosure.